### PR TITLE
Autoinjector, Hypospray, and assiocated pouches balance 

### PR DESCRIPTION
**Pull request by Alterist**
## About The Pull Request
Buffs Combat Autoinjector to 10/10/5u Bica/Kelo/Tram
Changes Advanced Combat Autoinjector to 2 shots of 10/10/10u mera/derma/tram
buffs Oxycodone Autoinjector to 2 shots of 10u 
Adds polyhex and Ironutriment Autoinjectors
Autoinjector pouch now hold 8 
Synth Closet Autoinjector Pouch now contains a variety of unique and useful chems
Corpsman Closet Autoinjector Pouch now contains some more useful chems

## Why It's Good For The Game

more choices for medical, life good. also why do medics get a hypo that purges part of itself

## Changelog
:cl:
add: Larvaway and Ironutriment Autoinjectors now exists
balance: Combat Autoinjector to 10/10/5u Bica/Kelo/Tram
balance: Advanced Combat Autoinjector to 2 shots of 10/10/10u mera/derma/tram
balance: Oxycodone Autoinjector to 2 shots of 10u 
balance: Autoinjector pouch can now hold 8 autoinjectors
balance: Synth Closet Autoinjector Pouch now contains a variety of unique and useful chems
balance: Corpsman Closet Autoinjector Pouch now contains some more useful chems
balance: Corpsman Hypo pouch now holds 5
balance: Corpsman hypo pouch now comes with both a meraderma hypo and an oxy hypo.
/:cl:

### CODE REVIEW

**Comment by Swagile** [comment]
there's a reason you don't get oxy in advanced injectors

dermaline and meralyn instantly delete any oxycodone detected in your system

if you want to buff advanced autoinjectors, give them hydrocodone, or whatever the full name is that is located in the B18's autodoc system

File: code/game/objects/items/reagent_containers/autoinjectors.dm [hunk 1]
```diff
@@ -206,6 +206,24 @@
 		/datum/reagent/medicine/hyronalin = 6,
 	)
 
+/obj/item/reagent_containers/hypospray/autoinjector/larvaway
+	name = "tramadol autoinjector"
```
Comment (Swagile) [inline code/game/objects/items/reagent_containers/autoinjectors.dm:210]: Name = "larvaway autoinjector"

Comment (Swagile) [inline code/game/objects/items/reagent_containers/autoinjectors.dm:210]: fix name mabe?
```suggestion
	name = "larvaway autoinjector"
```

**Comment by Alterist** [comment]
This should have been a draft, will reopen when complete

File: code/game/objects/items/reagent_containers/autoinjectors.dm (hunk 1 above)
Comment (Alterist) [reply code/game/objects/items/reagent_containers/autoinjectors.dm]: fixed

**Comment by SplinterGP** [comment]
autoinjector pouch doenst need to hold more.

**Comment by Alterist** [comment]
@SplinterGP they do, currently there's absolutely no reason you would pick them from the  as a synth or corpsman, and even less reason to use them as marine, as Medkit pouches does everything they do but better

File: code/game/objects/items/reagent_containers/autoinjectors.dm [hunk 2]
```diff
@@ -190,34 +193,54 @@
...
 	volume = 9
-	icon_state = "autoinjector-8"
 	list_reagents = list(
 		/datum/reagent/medicine/peridaxon_plus = 3,
 		/datum/reagent/medicine/hyronalin = 6,
 	)
 
+/obj/item/reagent_containers/hypospray/autoinjector/polyhexanide
+	name = "Larvaway autoinjector"
+	desc = "An auto-injector loaded with a dose of Polyhexanide, a sterilizer for internal surgical use."
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/autoinjectors.dm:214]: larvaway?

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by JackTheJackhammer** [comment]
Definitely a needed change. Its horrible having to stick 2 needles to even get myself in a position to start healing

File: code/game/objects/items/reagent_containers/autoinjectors.dm (hunk 2 above)
Comment (Alterist) [reply code/game/objects/items/reagent_containers/autoinjectors.dm]: fixed

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**